### PR TITLE
Add new setting queue_size inside whodata configuration block

### PR DESCRIPTION
**Pull request by jotacarma90**
## Description
This PR is to add the new setting queue_size inside the whodata configuration block, syscheck.

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Review by chemamartinez — APPROVED** [review]
LGTM

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 1]
```diff
@@ -1010,6 +1011,19 @@ Allows disabling the Audit health check during the Whodata engine starting. This
...
+
+Sets the maximum capacity of the queue that stores the audit dispatcher events. **Linux systems with Audit**.
+
++--------------------+---------------------------------+
+| **Default value**  | 16384                           |
++--------------------+---------------------------------+
+| **Allowed values** | Any number from 10 to 1048576   |
++--------------------+---------------------------------+
+
+.. warning:: If the queue becomes full, some audit events may be lost. However, the next scheduled scan will generate the missing alerts, without the audit information.
```
Comment (s-ocando) [inline source/user-manual/reference/ossec-conf/syscheck.rst:1025]:
```suggestion
.. warning:: If the queue fills up, some audit events may be lost. However, the next scheduled scan will generate the missing alerts without the audit information.
```

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 2]
```diff
@@ -1010,6 +1011,19 @@ Allows disabling the Audit health check during the Whodata engine starting. This
 
 .. warning:: The health check ensures that the rules required by Whodata can be set in Audit correctly and also that the generated events can be obtained. Disabling the health check may cause functioning problems in Whodata and loss of FIM events.
 
+
+**queue_size**
+
+Sets the maximum capacity of the queue that stores the audit dispatcher events. **Linux systems with Audit**.
```
Comment (s-ocando) [inline source/user-manual/reference/ossec-conf/syscheck.rst:1017]:
```suggestion
Sets the maximum capacity of the queue that stores the audit dispatcher events. This option applies to **Linux systems with Audit**.
```

**Review by s-ocando — COMMENTED** [review]
Hi @jotacarma90, please review my suggestions below.

File: source/user-manual/reference/ossec-conf/syscheck.rst (hunk 1 above)
Comment (jotacarma90) [reply source/user-manual/reference/ossec-conf/syscheck.rst]: Done

File: source/user-manual/reference/ossec-conf/syscheck.rst (hunk 2 above)
Comment (jotacarma90) [reply source/user-manual/reference/ossec-conf/syscheck.rst]: Done

**Pull request merged**